docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -1,90 +1,201 @@
+access to your infrastructure.
 
-## Step 1/5 Sign up
+Teleport Enterprise Cloud includes two components:
 
-Sign up for a cloud account [here](https://goteleport.com/signup/).
+- **Teleport Auth Service:** The certificate authority for your cluster. It
+  issues certificates and conducts authentication challenges.
+- **Teleport Proxy Service:** The cluster frontend, which handles user requests,
+  forwards user credentials to the Auth Service, and communicates with Teleport
+  instances that enable access to specific resources in your infrastructure.
 
-## Step 2/5 Access Web Console
+In this guide, you will register a server with your Teleport Cloud account so
+you can access it securely. Your server will connect to the Teleport Proxy
+Service and open a reverse tunnel that the Proxy Service can use to allow
+traffic to the server from authorized clients.
 
-Access Web Console
-![Access Web Console](../../../img/cloud/homepage-1.png)
+## Prerequisites
 
-Select Add Server and press COPY to copy the script command
-![Add Server Script](../../../img/cloud/addserver-2.png)
+- A Teleport Enterprise Cloud account. You can sign up for a free trial at the
+  following page:
 
-## Step 3/5 Install Teleport Agent on Server
+  https://goteleport.com/signup/
 
-Paste and run the script on a Linux Server:
+- Docker installed on your workstation. We will use a Docker container to
+  register a server with Teleport via the Teleport SSH Service. You can find
+  installation instructions on [Docker's
+  website](https://docs.docker.com/get-docker/).
 
-![Past Install Script](../../../img/cloud/runscript-3.png)
+  <Notice type="tip">
 
-Teleport Agent Installed
-![Completed Install](../../../img/cloud/completed-script-4.png)
+  Docker is only required for a local demo environment. If you want to get
+  started registering your servers in your infrastructure with Teleport, view
+  our documentation on how to [register servers and configure access to
+  them](../../server-access/introduction.mdx).
 
-## Step 4/5 Access Server
+  </Notice>
 
-Click **CLOSE**.  The server can now be accessed via SSH:
-![Access Server](../../../img/cloud/online-session-5.png)
+- The `tsh` client tool. Visit our [Installation Guide](../../installation.mdx)
+  for instructions. If you want to get started quickly, you can skip this for
+  now and access Teleport through your browser.
 
-Interact with your SSH session:
-![Open SSH](../../../img/cloud/online-ssh-5.png)
+## Step 1/4. Spin up your server
 
-Exit the session with `exit`. You can replay the session within Session Recordings:
-![Replay Session](../../../img/cloud/online-replay-5.png)
+Spin up a server on your workstation. In Step 2, we will register this server
+with your Teleport Enterprise Cloud tenant:
 
-## Step 5/5 Access from Command Line
+```code
+$ docker run -it ubuntu:22.10 /bin/bash
+```
+
+In the shell you opened within the `ubuntu:22.10` container, make sure you have
```

Comment:
```suggestion
This will bring your terminal session into the `ubuntu:22.10` container. From there, make sure you have
```

Reply:
Doesn't have to be this specifically, I just wanted to change the language to explain that the same session you're already in is now operating from a different environment.